src/runtime/basic-runtime.js

```diff
@@ -277,6 +278,10 @@ export class ConfiguredBasicRuntime {
     // Fetch the client config.
     this.configuredClassicRuntime_.clientConfigManager().fetchClientConfig();
 
+    // Start listening to Google Analytics events.
+    this.googleAnalyticsEventListener_ = new GoogleAnalyticsEventListener(this);
```

Comment:
Could you annotate this variable with jsdoc?

Reply:
Done.